actions/apt_action.go

```diff
@@ -44,7 +44,6 @@ func NewAptAction() *AptAction {
 }
 
 func (apt *AptAction) Run(context *debos.DebosContext) error {
```

Comment:
missing for arch actions ;-)

Reply:
Correct - there is a big NOTE that aims to highlight that

Reply:
right I only looked at the commit description & contents.